.github/pull_request_template.md

```diff
@@ -3,5 +3,5 @@ Please always provide the [GitHub issue(s)](../issues) your PR is for, as well a
 Fix #<gh-issue-id>
```

Comment:
maybe use {gh-issue-id} as <> is hidden by default.

Reply:
It's only hidden in the markdown preview, it's fine once you open a PR